routes/api.go

```diff
@@ -53,9 +57,9 @@ func InitializeRoutes(c *gin.Engine, dbSession *mgo.Session, conf *config.Config
 	// privateEndpoints.DELETE("/organization", controllers.DeleteOrganization)
 
 	// // User
-	// privateEndpoints.GET("/users", controllers.GetUsers)
-	// privateEndpoints.GET("/users/:userId", controllers.GetUser)
-	// privateEndpoints.POST("/users", controllers.PostUser)
+	publicEndpoints.GET("/users", userApi.GetUsers)
+	//publicEndpoints.GET("/users/:userId", controllers.GetUser)
+	publicEndpoints.POST("/users", userApi.Create)
 	// privateEndpoints.PUT("/users", controllers.PutUser)
```

Comment:
Is the update user going to be implemented later?

Reply:
Yes.